runc_test.go

```diff
@@ -314,6 +314,23 @@ func dummySleepRunc() (_ string, err error) {
 	return fh.Name(), nil
 }
 
+func TestRuncCheckpoint(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	okRunc := &Runc{
+		Command: "/bin/true",
```

Comment:
I also tried `/bin/echo` thinking I could validate command would be echo'd back in output for better assertion; however, `runOrError` only returns output as error when non-zero exit code.